FiveCalls/FiveCalls/IssuesContainerViewController.swift

```diff
@@ -65,13 +62,56 @@ class IssuesContainerViewController : UIViewController, EditLocationViewControll
     }
 
     func editLocationViewController(_ vc: EditLocationViewController, didSelectZipCode zip: String) {
+        dismiss(animated: true, completion: { //`updateWith` must be in completion block so that VC is listening for notification
+            self.updateWith(zipCode: zip)
```

Comment:
Should use [weak self] on completion blocks that reference self. (A few of them here below as well)